Makefile

```diff
@@ -64,6 +98,18 @@ build_event-bus: build_shared_package_container
 server_ci: start_network
 	make lint_server test_server push_server_container
 
+server_if_needed:
+ifneq "$(strip $(filter server/%, $(shell git diff --name-only master...${TRAVIS_BRANCH})))" ""
+	make server_ci
+else ifeq ("${TRAVIS_BRANCH}", "master")
+ifeq ("${TRAVIS_PULL_REQUEST}", "false")
+	@echo "working on master building everything"
+	make server_ci
+else
+	@echo "this is a pr, no action necessary"
```

Comment:
the statuses reported at https://github.com/libero/reviewer/pull/267/checks don't look very good because they will be green even if master is currently broken. Ideal thing would be not to have the checks that are not relevant there, but I'm not aware of a way to make this dynamic in Travis CI. Doesn't seem well-suited to monorepos.

Reply:
I think this will expedite the conversation to split the repo tomorrow :)

Reply:
When?